jupyterlab/package.json

```diff
@@ -36,7 +36,7 @@
     "typescript": "^5.0.2"
   },
   "dependencies": {
-    "@codemirror/language": "6.6.0",
+    "@codemirror/language": "6.7.0",
     "@codemirror/legacy-modes": "6.3.2",
```

Comment:
Updated because `@jupyterlab/codemirror` also has a dependency on `@codemirror/language`, and when the versions don't match, the same types end up getting defined twice. Occasionally, they subtly conflict and TypeScript complains.

There were no errors before, because we were failing to resolve typings for these modules so everything was `any`.